test/mockTrip.js

```diff
@@ -68,27 +105,29 @@ export const mockFeatureCollection = (id, features) => ({
   features
 })
 
-export const mockSerie = (id = 'serieId01') => ({
+export const mockSerie = (id = 'serieId01', features) => ({
   id,
   type: 'FeatureCollection',
   properties: {
     start_place: { $oid: 'sectionId01' },
     end_place: { $oid: 'sectionId02' }
   },
-  features: [
+  features: features || [
     mockFeature('sectionId01'),
     mockFeature('sectionId02'),
     mockFeatureCollection('sectionId03', [mockFeature('featureId01')])
   ]
 })
 
-export const mockTimeserie = (id = 'timeserieId01', series) => ({
-  _id: id,
-  id,
-  _type: DOCTYPE_GEOJSON,
-  cozyMetadata: {},
-  startDate: '',
-  endDate: '',
-  source: '',
-  series
-})
+export const mockTimeserie = (id = 'timeserieId01', series) => {
+  return {
```

Comment:
let's revert this return adding 🙏

Reply:
j'ai l'avis contraire, pour tester c'est pénible sans

Reply:
*A débattre au prochain point, je ne pas pour ajouter ce genre de règle :)*